Patch migrated from Desktop, where it was listed as:
# date:2012-12-19 owner:Rohini S <rohini.s@oracle.com> type:bug

Solaris only patch - need to determine if we can find better fix

--- dbus-1.4.18/m4/libtool.m4-orig	2012-02-13 17:40:45.822749664 -0600
+++ dbus-1.4.18/m4/libtool.m4	2012-02-13 17:41:02.499016163 -0600
@@ -1471,13 +1471,13 @@ old_postuninstall_cmds=
 if test -n "$RANLIB"; then
   case $host_os in
   openbsd*)
-    old_postinstall_cmds="$old_postinstall_cmds~\$RANLIB -t \$tool_oldlib"
+    old_postinstall_cmds="$old_postinstall_cmds~\$RANLIB -t \$oldlib"
     ;;
   *)
-    old_postinstall_cmds="$old_postinstall_cmds~\$RANLIB \$tool_oldlib"
+    old_postinstall_cmds="$old_postinstall_cmds~\$RANLIB \$oldlib"
     ;;
   esac
-  old_archive_cmds="$old_archive_cmds~\$RANLIB \$tool_oldlib"
+  old_archive_cmds="$old_archive_cmds~\$RANLIB \$oldlib"
 fi
 
 case $host_os in
